src/entities/RfqmJobEntity.ts

```diff
@@ -10,11 +10,36 @@ export interface StoredFee {
 }
 
 export enum RfqmJobStatus {
-    InQueue = 'inQueue',
-    Processing = 'processing',
-    Submitted = 'submitted',
-    Successful = 'successful',
-    Failed = 'failed',
+    // Transaction has been enqueued and will be processed once a worker is available
+    PendingEnqueued = 'pending_enqueued',
+    // Transaction has passed initial validation. Last look will be executed and transaction will be submitted if last look is accepted.
+    PendingProcessing = 'pending_processing',
+    // Transaction has passed initial verification and has been submitted to the mem pool
+    PendingSubmitted = 'pending_submitted',
+
+    // Eth Call made before transaction submission was unsuccessful
+    FailedEthCallFailed = 'failed_eth_call_failed',
+    // Transaction has expired prior to eth call or worker is not available to make an eth call
+    FailedExpired = 'failed_expired',
+    // Market Maker declined the last look
+    FailedLastLookDeclined = 'failed_last_look_declined',
+    // Transaction was reverted
+    FailedReverted = 'failed_reverted',
+    // Submitting the transaction to the network was unsuccessful
+    FailedSubmitFailed = 'failed_submit_failed',
+    // Transaction does not contain call data
+    FailedValidationNoCallData = 'failed_validation_no_call_data',
```

Comment:
slight pref for collapasing these 4 into FailedJobValidationError

Reply:
ack. i think since it's here i'll leave it. also if the code starts going rogue and not making valid orders it'll be easier to debug.